src/main/java/com/intuit/graphql/orchestrator/datafetcher/ServiceDataFetcher.java

```diff
@@ -7,16 +7,18 @@
 import lombok.Getter;
 
 @Getter
-public class ServiceDataFetcher implements DataFetcher {
+public class ServiceDataFetcher implements DataFetcher, ServiceContext {
```

Comment:
I am curious about this design pattern, given the fact that the serviceMetadata that has serviceProvider that has namespace

Reply:
We can possibly do 
```
NamespaceAwareDataFetcher implements DataFetcher {
...
}
```

and all DF extend this one instead DataFetcher directly. you can also do ServiceAwareDF if all DF has that information

Reply:
That is a good idea, i will update.